packages/menu/src/Menu.story.js

```diff
@@ -1,56 +1,60 @@
 import React, { useState } from 'react';
 import { storiesOf } from '@storybook/react';
 import { select, boolean } from '@storybook/addon-knobs';
-
+import InteractionContext from '@leafygreen-ui/interaction-context';
 import { Menu, MenuGroup, MenuItem } from './index';
 import { Align, Justify } from '@leafygreen-ui/popover';
 
 function Uncontrolled() {
   return (
-    <Menu
-      align={select('Align', Object.values(Align), Align.Bottom)}
-      justify={select('Justify', Object.values(Justify), Justify.Start)}
-      trigger={<button>trigger</button>}
-    >
-      <MenuItem active>Active Menu Item</MenuItem>
-      <MenuItem
-        disabled={boolean('Disabled', true)}
-        description="I am a description"
+    <InteractionContext>
+      <Menu
+        align={select('Align', Object.values(Align), Align.Bottom)}
+        justify={select('Justify', Object.values(Justify), Justify.Start)}
+        trigger={<button>trigger</button>}
       >
-        Disabled Menu Item
-      </MenuItem>
-      <MenuItem description="I am also a description">
-        Menu Item With Description
-      </MenuItem>
-      <MenuItem href="http://mongodb.design">I am a link!</MenuItem>
-    </Menu>
+        <MenuItem active>Active Menu Item</MenuItem>
+        <MenuItem
+          disabled={boolean('Disabled', true)}
+          description="I am a description"
+        >
+          Disabled Menu Item
+        </MenuItem>
+        <MenuItem description="I am also a description">
+          Menu Item With Description
+        </MenuItem>
+        <MenuItem href="http://mongodb.design">I am a link!</MenuItem>
+      </Menu>
+    </InteractionContext>
   );
 }
 
 function Controlled() {
   const [open, setOpen] = useState(false);
   return (
-    <button onClick={() => setOpen(!open)}>
-      trigger
-      <Menu
-        align={select('Align', Object.values(Align), Align.Bottom)}
-        justify={select('Justify', Object.values(Justify), Justify.Start)}
-        open={open}
-        setOpen={setOpen}
-      >
-        <MenuGroup>
-          <MenuItem active>Active Menu Item</MenuItem>
-          <MenuItem disabled={boolean('Disabled', true)}>
-            Disabled Menu Item
-          </MenuItem>
-          <MenuItem description="I am a description">
-            Menu Item With Description
-          </MenuItem>
-          <MenuItem href="http://mongodb.design">I am a link!</MenuItem>
-        </MenuGroup>
-        <MenuItem>Left out of the MenuGroup</MenuItem>
-      </Menu>
-    </button>
+    <InteractionContext>
```

Comment:
If we keep on this trend of having global context providers we may want to have one `LeafygreenProvider` in the future that wraps them all up for us. Don't need it now, but just an idea.

Reply:
That's the idea for this, and why we didn't just export `UsingKeyboardContext.Provider` instead. We can add context providers in the InteractionContext component. If you have better ideas for this name, let me know (aside from renaming to `InteractionProvider`).

Reply:
I know @brookescarlett wasn't a fan of `LeafyGreenProvider` initially

Reply:
I wasn't at first, because I didn't think of it as a wrapper. I support having one wrapper called LeafyGreen provider.

Reply:
Done.